fe/fe-core/src/main/java/com/starrocks/analysis/BaseGrantRevokeRoleStmt.java

```diff
@@ -0,0 +1,44 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import com.starrocks.sql.ast.AstVisitor;
+
+// GrantRoleStmt and RevokeRoleStmt share the same parameter and check logic with GrantRoleStmt
+public class BaseGrantRevokeRoleStmt extends DdlStmt {
```

Comment:
new statement should under /starrocks/sql/ast

Reply:
OK